include/materials/mg_nuclide.hpp

```diff
@@ -45,7 +45,8 @@ class MGNuclide : public Nuclide {
             const std::vector<std::vector<MGAngleDistribution>>& angle,
             const std::vector<double>& P_dlyd_grp,
             const std::vector<double>& decay_cnsts,
-            const std::vector<std::vector<double>>& delayed_chi);
+            const std::vector<std::vector<double>>& delayed_chi,
+            const std::vector<double>& Eh);
```

Comment:
No need for this change in multi-group mode. Just implement a "heating" method which always returns 0. If we need heating in MG mode one day, we will add it then. Until that point, I don't want to add the complexity.